package.json

```diff
@@ -2,21 +2,19 @@
   "name": "@chainsafe/blst",
   "version": "0.2.10",
   "description": "Typescript wrapper for supranational/blst native bindings, a highly performant BLS12-381 signature library",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "files": [
-    "dist",
-    "prebuild",
-    "blst"
-  ],
   "scripts": {
-    "bootstrap": "yarn install --ignore-scripts && yarn build && yarn install",
-    "build": "tsc --project tsconfig.build.json",
+    "install": "ts-node scripts/install",
```

Comment:
This will be refactored a bit in an upcoming PR to remove the hard dependency on ts-node but without requiring a “bootstrap” command that first builds the TS when working in the repo so a JS install.js is available.